### ISSUE TITLE
Settings a11y - Google play and App store images need alt text + darkened grey text

### ISSUE THREAD

**Issue by LZoog** [mozilla/fxa#10763]
When using a screenreader, only "Link, opens in new window" is read.

In `ConnectAnotherDevicePromo`, let's add alt text matching what's on the images. Don't forget to localize it.

Also, the `text-grey` class used on the "Find Firefox in the Google Play and App Store or" doesn't pass ADA AA against the white background. It should be `text-grey-400`.

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/13018240/137959937-24e2d304-473a-4e08-80dd-ab14fabf295f.png">

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-4127)
